### ISSUE TITLE
ssh chroot error when deployed in a podman pod

### ISSUE THREAD

**Issue by dchats1** [go-gitea/gitea#24618]
### Description

I recently tried setting up Gitea using Podman (v4.4.2) on Fedora Server 37. When I try to clone a repo I get the following error:
```
$ git clone git@gitea.lab.davidchatterton.com:dchatterton/test.git
Cloning into 'test'...
Connection closed by 172.16.20.70 port 22
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

And in the container logs I see the follow error every time I try to clone a repo:
```
chroot("/var/empty"): Operation not permitted [preauth]
```

I've tried with and without selinux enabled. I've also come across a couple related issues, but I haven't been able to come up with a fix for podman:
https://github.com/go-gitea/gitea/issues/15907
https://gitea.com/gitea/helm-chart/issues/8

### Gitea Version

1.19

### Can you reproduce the bug on the Gitea demo site?

No

### Operating System

Fedora Server 37, Podman 4.4.2

### How are you running Gitea?

The current deployment is as follows:
Podman Pod:
```
     "CreateCommand": [
          "podman",
          "pod",
          "create",
          "--name",
          "gitea-pod",
          "--network",
          "prod",
          "--ip",
          "172.16.20.70",
          "--hostname",
          "gitea.lab.davidchatterton.com"
     ],
```
and the container:
```
     "CreateCommand": [
          "/usr/bin/podman",
          "run",
          "--cidfile=/run/container-gitea.service.ctr-id",
          "--cgroups=no-conmon",
          "--rm",
          "--sdnotify=conmon",
          "--replace",
          "--detach",
          "--name",
          "gitea-test",
          "--pod",
          "gitea-pod",
          "-v",
          "gitea:/data:z",
           "docker.io/gitea/gitea:latest"
      ],
```

### Database

SQLite

**Comment by dchats1** [comment]
Thanks for the quick reply @wxiaoguang! The `--cap-add=SYS_CHROOT` flag fixed the issue!

**Comment by wxiaoguang** [comment]
I think we can keep this issue open, to improve documents.

**Comment by gorsing** [comment]
Fix in podman 4.5.1
